### DISCUSSION TITLE
Which should be required fields

### DISCUSSION THREAD

**Discussion by callummole** [alan-turing-institute/AutSPACEs#400] [General]
Currently there are three required fields, without which a participant will not be able to share a story on autspaces (see [comment](https://github.com/alan-turing-institute/AutSPACEs/pull/394#discussion_r1045625713)). These are:

- title
- experience
- suggestions

All the checkboxes are currently optional (i.e. False if not checked). 

Which fields do we want to be essential?

**Comment by callummole** [comment]
my suggestion is that we have the title and experience as required, but suggestion as optional.

**Comment by gedankenstuecke** [comment]
I agree, having the Title & experience itself as required and all the rest optional sounds like a good approach to me!
